### PR TITLE
fix: file name with ampersand error

### PR DESCRIPTION
**Pull request by banusha2**
## Context

If an artifact has a filename with an ampersand in it,UI shows an error .
## Objective

Screen Shot 2023-02-24 at 11 30 21 AM

## References

https://github.com/screwdriver-cd/screwdriver/issues/2839
## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Review by adong — CHANGES_REQUESTED** [review]
This feature is still under testing.

**Comment by adong** [comment]
image

File: app/components/artifact-tree/component.js [hunk 1]
```diff
@@ -79,7 +79,8 @@ export default Component.extend({
 
     showArtifactPreview(treeData) {
       const { href } = treeData.a_attr;
-      const artifactPath = href.split('artifacts/')[1];
+      const pathArray = href.split('artifacts/');
+      const artifactPath = pathArray[pathArray.length - 1];
```
Comment (adong) [inline app/components/artifact-tree/component.js:83]: Still does not work for me

Comment (banusha2) [reply app/components/artifact-tree/component.js:83]: What is not working? means you cannot open file with ampersand in it?

Comment (adong) [reply app/components/artifact-tree/component.js:83]: Correct. I test branch with internal production, and cannot open artifacts with this url

```
<screwdriver-url>/pipelines/1015452/builds/34724462/artifacts/cypress/screenshots/common/dashboards/performance.js/Performance%20Dashboard%20tests%20--%20App%20&%20Website%20tests%20--%20should%20show%20data%20after%20switching%20to%20UTC%20timezone%20(failed)%20(attempt%202).png
```

**Comment by adong** [comment]
It turns out not to be a UI issue, closed.

**Pull request closed without merging**